starboard/android/shared/media_codec_bridge.cc

```diff
@@ -180,7 +180,17 @@ std::unique_ptr<MediaCodecBridge> MediaCodecBridge::CreateAudioMediaCodecBridge(
 
   if (MediaCodecBridgeEradicator::GetInstance()->IsEnabled()) {
     // block if the old MediaCodecBridge instances haven't been destroyed yet
-    MediaCodecBridgeEradicator::GetInstance()->WaitForPendingDestructions();
+    bool signaled =
```

Comment:
nit: We shouldn't use "signaled" as variable name here, as using ConditionVariable is an implementation detail that's not visible here.

Reply:
renamed